static/app/components/alerts/snoozeAlert.tsx

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled';
+
+import {Client} from 'sentry/api';
+import {Button} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {DropdownMenu, MenuItemProps} from 'sentry/components/dropdownMenu';
+import {IconChevron, IconMute, IconSound} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {Organization} from 'sentry/types';
+import withApi from 'sentry/utils/withApi';
+import withOrganization from 'sentry/utils/withOrganization';
+
+type Props = {
+  api: Client;
+  isSnoozed: boolean;
+  organization: Organization;
+  projectId: string;
+  ruleId: string;
+  setSnooze: (nextState: boolean) => void;
+  setSnoozeCreatedBy: (nextState: string) => void;
+};
+
+function SnoozeAlert(props: Props) {
+  const {isSnoozed, setSnooze, setSnoozeCreatedBy, api, organization, projectId, ruleId} =
+    props;
```

Comment:
nit - just destructure the props in arguments if you're going to destructure them on the next line
```suggestion
function SnoozeAlert({isSnoozed, setSnooze, setSnoozeCreatedBy, api, organization, projectId, ruleId} : Props) {
```